addons/sale/wizard/res_config_settings.py

```diff
@@ -74,6 +74,7 @@ class ResConfigSettings(models.TransientModel):
     module_product_email_template = fields.Boolean("Specific Email")
     module_sale_amazon = fields.Boolean("Amazon Sync")
     module_sale_commission = fields.Boolean("Commissions")
+    module_sale_gelato = fields.Boolean("Gelato")
```

Comment:
Why not after Amazon if it's the case in the view?

Reply:
The setting file seems to follow more an alphabetic order rather than view order so I was following that to not introduce irregularity. In view file it is module_sale_commission and then module_sale_amazon and then in this file module_sale_amazon and then module_sale_commison.